recipes/_omnibus_toolchain.rb

```diff
@@ -56,5 +56,6 @@
   end
 end
 
+certs_path = ::File.join(toolchain_install_dir, 'embedded', 'ssl', 'certs', 'cacert.pem')
 omnibus_env['OMNIBUS_TOOLCHAIN_INSTALL_DIR'] << toolchain_install_dir
-omnibus_env['SSL_CERT_FILE'] << windows_safe_path_join(toolchain_install_dir, 'embedded', 'ssl', 'certs', 'cacert.pem')
+omnibus_env['SSL_CERT_FILE'] << (windows? ? windows_safe_path_join(certs_path) : certs_path)
```

Comment:
This change seems redundant now that `windows_safe_path_join ` has been tweaked. `windows_safe_path_join` should equally well (and as expected) on Windows and *nix platforms.